inspirehep/modules/workflows/tasks/arxiv.py

```diff
@@ -133,12 +135,22 @@ def arxiv_plot_extract(obj, eng):
             current_app.logger.exception(err)
             return
 
+        lb = LiteratureBuilder(source=obj.data['acquisition_source']['source'], record=obj.data)
         for idx, plot in enumerate(plots):
+            key = '{uuid}_{name}'.format(uuid=uuid.uuid4(), name=plot.get('name'))
```

Comment:
do we really need a UUID here? it seems a bit overkill and will result in very long file names.